<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>160</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="paramLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>a:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="aEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>b:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bEdit">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>m:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mEdit">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>l:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lEdit">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="problemInfoLayout">
     <item>
      <widget class="QLabel" name="problemNameLabel">
       <property name="text">
        <string>Calculation of function integral. Podkopaev Anton, 345 group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="functionLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>741</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Middle rectangle</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Trapezium</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Simpson</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>S_m</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|Integral - Sm|</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S_lm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|Integral - S_lm|</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>I_AD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>|Integral - I_AD|</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>I_AD - integral accurancy definition (with Runge correction)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
